src/docfx/build/toc/TableOfContentsItem.cs

```diff
@@ -2,37 +2,38 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Docs.Build
 {
     public class TableOfContentsItem
     {
-        [JsonProperty(PropertyName = "toc_title")]
```

Comment:
does this need to be `SnakeCaseNamingStrategy`? since it was `toc_title`

Reply:
the other properties like `tocHref` are not using `SnakeCase`.